edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Core/ServiceIdentityDictionary.cs

```diff
@@ -25,7 +25,7 @@ public ServiceIdentityDictionary(string actorDeviceId)
 
         public string GetActorDeviceId() => this.actorDeviceId;
 
-        public Task<Option<string>> GetAuthChain(string id) => throw new NotImplementedException("Nested Edge not enabled");
+        public Task<Option<string>> GetAuthChain(string id) => Task.FromResult(Option.None<string>());
 
         public Task<Option<string>> GetEdgeAuthChain(string id) => throw new NotImplementedException("Nested Edge not enabled");
```

Comment:
Could a similar problem happen due to these related methods throwing?

Reply:
I don't think so since they're not called anywhere else in non-nested scenarios